fdbserver/fdbserver.actor.cpp

```diff
@@ -1703,6 +1704,10 @@ struct CLIOptions {
 				knobs.emplace_back("kms_connector_type", args.OptionArg());
 				break;
 			}
+			case OPT_KMS_REST_ENABLE_NOT_SECURE_CONECTION: {
+				knobs.emplace_back("rest_kms_enable_not_secure_connection", args.OptionArg());
```

Comment:
Since this guards a pretty sensitive operation i feel like we should at least log a warning if this knob is turned on through the CLI

Reply:
Sounds like a good idea. Shall do it. 
Also, RESTUtils does a sanity check on this Knob and generates appropriate error log; but an `info` log here is helpful.